spec/unit/combo-box/combo-box.spec.js

```diff
@@ -500,4 +500,20 @@ describe("combo box component", () => {
     assert.ok(list.hidden, "should hide the option list");
     assert.strictEqual(document.activeElement, input, "should focus the input");
   });
+
+  it("should not allow for innerHTML of child elements ", () => {
+    input.value = "apricot";
+
+    EVENTS.input(input);
+    assert.ok(!list.hidden, "should display the option list");
+
+    // eslint-disable-next-line no-plusplus
+    for (let i = 0; i < list.children.length; i++) {
+      assert.strictEqual(
+        list.children[i].children.innerHTML,
+        undefined,
+        "should not contain child HTML"
+      );
+    }
```

Comment:
`children.innerHTML` will always be `undefined`, because [`children` is an array-like `HTMLCollection`](https://developer.mozilla.org/en-US/docs/Web/API/Element/children).

You may need a nested loop to check each list item's children. As an alternative to checking `innerHTML`, you could check that each of the children is a text node:

```suggestion
  list.children.forEach((listItem) => {
    listItem.children.forEach((childNode) => {
      assert.strictEqual(childNode.nodeType, Node.TEXT_NODE);
    });
  });
```

Reply:
TIL!! this change worked for us locally - committing the suggestion

Reply:
Hm, something's not quite right with this actually. That assertion should be checking `childNode.nodeType`. In debugging, it looks like the assertion's not being called, because `Array.from(listItem.children)` is empty, even though that's valid in the real-world 🤷  Might need to revert back to plain-ol' `for` loop?

Reply:
oof, this thing is trickier than I expected, taking another gander

Reply:
Ah, I see the issue. `children` doesn't include the text nodes ([source](https://developer.mozilla.org/en-US/docs/Web/API/Element/children)). Works after switching to `childNodes`.

```js
    Array.from(list.children).forEach((listItem) => {
      Array.from(listItem.childNodes).forEach((childNode) => {
        assert.strictEqual(childNode.nodeType, Node.TEXT_NODE);
      })
    });
```

Reply:
confirming that [nodeType](https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType#node_type_constants)  is returning  `3`

<img width="815" alt="Screen Shot 2021-08-12 at 5 05 08 PM" src="https://user-images.githubusercontent.com/37077057/129269905-9d29f1cf-e948-430a-b7ef-4ba5415b3c7e.png">